src/pages/swap/index.tsx

```diff
@@ -177,6 +182,7 @@ export const SwapPage = () => {
           outputTokenType: to as OutputTokenType,
           amountInUnits: fromAmountString,
           offrampAmount: tokenOutAmountData.roundedDownQuotedAmountOut,
+          setInitializeFailed,
```

Comment:
I realized we had no action for when `handleOnSubmit` failed. Probably this doesn't happen to often since `sep-10` is quite reliable, but now we add the message sign in the middle which could fail from the s. service.

With this, we will just show the `Application initialization fail...` message in this case. Same as on the pre-flight check. Let me know what you think.